services/spar/test/Test/Spar/ScimSpec.hs

```diff
@@ -85,3 +102,124 @@ spec = describe "toScimStoredUser'" $ do
 
     Scim.meta result `shouldBe` meta
     Scim.value (Scim.thing result) `shouldBe` usr
+
+  it "roundtrips" . property $ do
+      \(sue :: ScimUserExtra) ->
+        eitherDecode' (encode sue) `shouldBe` Right sue
+
+  describe "ScimUserExtra" $ do
+    describe "Patchable" $ do
```

Comment:
Maybe a test that shows attributes are case-insensitive? Though I hope that test is already in the `CI` datatype in the library we depend on...

Reply:
Done :)